test/celeritas/field/FieldPropagator.test.cc

```diff
@@ -693,7 +693,7 @@ TEST_F(TwoBoxTest, electron_tangent_cross)
         if (!CELERITAS_USE_VECGEOM)
         {
             EXPECT_EQ("inner_box.py",
-                      this->geometry()->id_to_label(geo.surface_id()));
+                      this->geometry()->id_to_label(geo.surface_id()).name);
```

Comment:
It might be worth adding a `surface_name(geo)` test fixture accessor like the `volume_name` on line 699, for brevity.

Reply:
Looks it already exists, but wasn't being used for some reason